src/main/resources/swagger/api-docs.yaml

```diff
@@ -8188,6 +8188,8 @@ components:
             - Creating
             - Ready
             - Error
+            - Deleting
```

Comment:
Noticed these were missing and updated in my open Rawls PR as well--

https://github.com/broadinstitute/rawls/pull/2280/commits/4b492606b2085d2b8767bf807f8c0f159a4baee7